src/game/gameflow.c

```diff
@@ -1299,3 +1299,114 @@ GameFlow_InterpretSequence(int32_t level_num, GAMEFLOW_LEVEL_TYPE level_type)
 
     return ret;
 }
+
+GAMEFLOW_OPTION
+GameFlow_StorySoFar(
```

Comment:
This function return value is not used. Let's make it a void.

Reply:
The return value `gf_option ` is used in `Savegame_PlayAvailableStory` for `gf_direction ` and `gf_param `:
```
    while (loop_continue) {
        int32_t gf_direction = gf_option & ~((1 << 6) - 1);
        int32_t gf_param = gf_option & ((1 << 6) - 1);

        gf_option = GameFlow_StorySoFar(gf_param, savegame_info->level_num);
        ...
```